src/codec/RowReaderWrapper.cpp

```diff
@@ -107,7 +107,7 @@ bool RowReaderWrapper::reset(meta::SchemaProviderIf const* schema,
     currReader_ = &readerV2_;
     return true;
   } else {
-    LOG(ERROR) << "Unsupported row reader version " << readerVer;
+    LOG(WARNING) << "Unsupported row reader version " << readerVer;
```

Comment:
How to distinguish between warning and error？

Reply:
It depends on whether administrator need to be involved. ERROR definitely need to be checked, but WARNING is not mandatory.